modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -67,21 +58,32 @@ networking:
   - 172.30.0.0/16
 endif::network[]
 platform:
-  vsphere:
-    vcenter: your.vcenter.server
-    username: username
-    password: password
-    datacenter: datacenter
-    defaultDatastore: datastore
-    folder: folder
-    resourcePool: resource_pool <6>
-    diskType: thin <7>
-    network: VM_Network
-    cluster: vsphere_cluster_name <8>
+  vsphere: <5>
     apiVIPs: 
-      - api_vip
-    ingressVIPs: 
-      - ingress_vip
+      - 10.0.0.1
+    failureDomains: <6>
+    - <failure_domain_name>
```

Comment:
`- <failure_domain_name>`
should change to
`- name: <failure_domain_name>`

Reply:
Updated.